draft-pda-protocol.md

```diff
@@ -573,116 +609,281 @@ PA protocol. The structure of the leader share is similarly protocol specific:
 ~~~
 struct {
   PAProto proto;
-  select (PALeaderShare.proto) {
-    case prio: PrioLeaderShare;
-    case hits: HitsLeaderShare;
+  select (PALeaderInputShare.proto) {
+    case prio: PrioLeaderInputShare;
+    case hits: HitsLeaderInputShare;
   };
-} PALeaderShare;
+} PALeaderInputShare;
 ~~~
 
-Note that the leader share is sent not encrypted.
+Note that only the helper's share is encrypted, and that the leader share is
+sent in plaintext. (Confidentiality of the leader's share is provided by the
+server-authenticated secure channel over which the request is made.)
 
 The leader responds to well-formed requests to `[leader]/upload_finish` with
 status 200 and an empty body. Malformed requests are handled as described in
 {{pa-error-common-aborts}}.
 
-[TODO: Since we're running the protocol with multiple cohorts of aggregators, the
-collector needs to decide how to pick which cohort has the "correct" output.
-This might be the cohort with the largest batch of inputs. Figure this out once
-the collection is specified.]
+## Collect {#pa-collect}
+
+~~~~
+Collector     Leader           Helper
+  |  collect 1  |                |
+  +------------->                |
+  |             |  aggregate 1   |
+  |             <---------------->
+  |             |  ...           |
+  |             |  aggregate L   |
+  |             <---------------->
+  |             |  output share  |
+  |             <---------------->
+  <-------------+                |
+  |  ...        |                |
+  |  collect N  |                |
+  +------------->                |
+  |             |  aggregate 1   |
+  |             <---------------->
+  |             |     ...        |
+  |             |  aggregate L   |
+  |             <---------------->
+  |             |  output share  |
+  |             <---------------->
+  <-------------+                |
+  v             v                v
+~~~~
+{: #pa-collect-flow title="Flow of the collect process with N collect requests
+and L aggregate requests per collect request."}
+
+[TODO: Decide how the collector's request is authenticated.]
```

Comment:
```suggestion
[TODO: Decide if and how the collector's request is authenticated.]
```

Reply:
Why does a collector's request to get the aggregate output need to be authenticated? That seems like a deployment specific property.

Reply:
I don't think this is going to end up being deployment specific, but I'll concede the point for now.